Libraries/PeriphDrivers/Include/MAX32672/gpio.h

```diff
@@ -314,17 +314,6 @@ void MXC_GPIO_RegisterCallback(const mxc_gpio_cfg_t *cfg, mxc_gpio_callback_fn c
  */
 void MXC_GPIO_Handler(unsigned int port);
 
-/**
- * @brief      Set Voltage select for pins to VDDIO or VDDIOH
- *
- * @param      port   The GPIO port
- * @param[in]  vssel  VDDIO or VDDIOH to set the voltatge to
- * @param[in]  mask   Pins in the GPIO port that will be set to the voltage.
- * 
- * @return     #E_NO_ERROR if everything is successful. See \ref MXC_Error_Codes for the list of error codes.
- */
-int MXC_GPIO_SetVSSEL(mxc_gpio_regs_t *port, mxc_gpio_vssel_t vssel, uint32_t mask);
```

Comment:
Added the "API Change" label since we removed this function.  Before merging we should add a note about this to the PR.  It's an unsupported feature of the micro, but still good practice to highlight this and why we did it for the next release notes.